initfiles/componentfiles/configxml/dafilesrv.xsd

```diff
@@ -74,6 +74,13 @@
                                 </xs:appinfo>
                             </xs:annotation>
                         </xs:attribute>
+                        <xs:attribute name="allowWriteFlush" type="xs:boolean" use="optional" default="true">
```

Comment:
see other comments. Not needed if an expert option, and I think it is of that ilk.